chart/values.yaml

```diff
@@ -72,6 +72,7 @@ config:
     process_throttle_secs: 5
 
 image:
+  image: freqtradeorg/freqtrade
```

Comment:
`.image.image` sounds repetitive. How about we change this to `.image.base`?

Reply:
I picked this name based on a pattern that I found here: https://github.com/kubernetes/ingress-nginx/blob/65ce8cfbdf1bd095bbc50723472118396ac5cfe2/charts/ingress-nginx/values.yaml#L14

I'm fine with changing it to base, but that does mean that it might break the regular convention of naming this variable though. In case you're fine with that, then that change to `image.base` is in 65c62aa.

Reply:
I don't think that reference necessarily makes it a regular convention, so let's go with `.image.base`.

The other thing that's missing is bumping up the chart version, but that's a little less relevant, since we're not really publishing this anywhere yet, so I'm going to approve this PR, and I'll work on publishing the chart later.